radixdlt-java-common/src/main/java/com/radixdlt/identifiers/Naming.java

```diff
@@ -0,0 +1,27 @@
+/*
+ * (C) Copyright 2021 Radix DLT Ltd
+ *
+ * Radix DLT Ltd licenses this file to you under the Apache License,
+ * Version 2.0 (the "License"); you may not use this file except in
+ * compliance with the License.  You may obtain a copy of the
+ * License at
+ *
+ *  http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND,
+ * either express or implied.  See the License for the specific
+ * language governing permissions and limitations under the License.
+ */
+
+package com.radixdlt.identifiers;
+
+import java.util.regex.Pattern;
+
+public final class Naming {
```

Comment:
I'm not sure about moving this logic outside of `CMAtomOS`. It's the OS which makes the rules and not a "common" rule. If this needs to change this would be part of something like the `ForkConfiguration` so would rather have this be closer to RadixEngine.